### ISSUE TITLE
Unknown loss softmaxCrossEntropy

### ISSUE THREAD

**Issue by vmarkovtsev** [tensorflow/tfjs#656]
#### TensorFlow.js version

0.12.5

#### Browser version
N/A - Node.js 10.9

#### Describe the problem or feature request

```js
model.compile({optimizer: 'rmsprop', loss: 'softmaxCrossEntropy', metrics: ['accuracy']});
```

raises

```
Error: Unknown loss softmaxCrossEntropy
    at new ValueError (/home/vadim/Projects/tfjs_notebook/node_modules/@tensorflow/tfjs-layers/dist/errors.js:36:28)
    at Object.get (/home/vadim/Projects/tfjs_notebook/node_modules/@tensorflow/tfjs-layers/dist/losses.js:184:15)
    at Model.compile (/home/vadim/Projects/tfjs_notebook/node_modules/@tensorflow/tfjs-layers/dist/engine/training.js:406:41)
```

However,

```js
model.compile({optimizer: 'rmsprop', loss: tf.losses.softmaxCrossEntropy, metrics: ['accuracy']});
```

works.

`softmax_cross_entropy` also does not work.

**Comment by nsthorat** [comment]
Like keras, we call it `categoricalCrossEntropy`.

**Comment by bileschi** [comment]
Attempting to make the error messaging clearer for this.
https://github.com/tensorflow/tfjs-layers/pull/307

**Comment by bileschi** [comment]
Closing this issue as due to bad documentation / error messaging which has been improved.

@vmarkovtsev Thanks for filing and please feel free to re-open if the provided solution doesn't seem sufficient.

**Comment by vmarkovtsev** [comment]
Cool, thanks
